lib/authorized_persona/view_helpers.rb

```diff
@@ -2,9 +2,9 @@ module AuthorizedPersona
   module ViewHelpers
     def authorized_to?(action, resource)
       route = Rails.application.routes.named_routes[resource]
-      raise AuthorizedPersona::Error, "Unable to determine route for #{resource}" if route.nil?
+      raise Error, "Unable to determine route for #{resource}" if route.nil?
 
-      controller_class = (route.defaults[:controller].camelize + 'Controller').constantize
+      controller_class = (route.defaults.fetch(:controller).camelize + 'Controller').constantize
```

Comment:
__suggestion: (nitpick)__ Perhaps this is more habit, but I usually don't like using `String#+` when I can do variable interpolation or other approaches, eg:

```ruby
controller_class = "#{route.defaults.fetch(:controller).camelize}Controller").constantize
```

The `String#+` allocates a new string. And I realize "this is ruby" so string allocation is the least of our worries in terms of memory usage. Even still, I usually try to bias towards approaches that create as few temporary objects as possible.